tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -726,9 +726,6 @@ public function testCreateNoUrl()
             '/div'
         ];
         $this->assertHtml($expected, $result);
-
-        $result = $this->Form->create(false, ['action' => false]);
-        $this->assertHtml($expected, $result);
```

Comment:
Because `url=>false` always has and will still be responsible for this. And `action` is deprecated all-together.